site/docs/src/pipeline.md

```diff
@@ -46,11 +46,11 @@ The pipeline has several stages:
 
 ### Modify pipeline configuration
 
-The `pipeline/config.yaml` file specifies how original `EO` programs are transformed into initial `EO` programs.
-Transformation is necessary because some objects in original `EO` programs don't work.
+The `pipeline/config.yaml` file specifies how original `EO` programs are rewriteed into initial `EO` programs.
+Rewriteation is necessary because some objects in original `EO` programs don't work.
```

Comment:
All these (funny) changes in this file should be removed.

Reply:
sorry for that

Reply:
No problem. Just remove all changes to that file.